packages/filesystem/src/browser/file-tree/file-tree-widget.ts

```diff
@@ -11,18 +11,27 @@ import {
     ContextMenuRenderer, VirtualRenderer,
     TreeWidget, NodeProps, TreeProps, ITreeNode
 } from "@theia/core/lib/browser";
+import { ElementAttrs } from "@phosphor/virtualdom";
 import { DirNode, FileStatNode } from "./file-tree";
 import { FileTreeModel } from "./file-tree-model";
+import { FileSystem } from '../../common';
+import * as base64 from 'base64-js';
+import { ConfirmDialog } from "@theia/core/lib/browser";
+import { ILogger } from '@theia/core/lib/common';
 
 export const FILE_TREE_CLASS = 'theia-FileTree';
 export const FILE_STAT_NODE_CLASS = 'theia-FileStatNode';
 export const DIR_NODE_CLASS = 'theia-DirNode';
 export const FILE_STAT_ICON_CLASS = 'theia-FileStatIcon';
+export const DROPZONE_CLASS = 'theia-Dropzone';
```

Comment:
`theia-dropzone`

Reply:
Done

Reply:
If I put "theia-dropzone", it is not the same format as the other. ex: "theia-DirNode", "theia-FileStatIcon".. They use cameCase, so we should use "theia-DropZone", theia-ActiveDropZone"

Reply:
I pointed it because in css you call it `dropzone` in the lower case